packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1270,8 +1270,10 @@ idx.enter.otp.in.original.tab = Enter the OTP in your original authentication lo
 idx.return.link.otponly.accessing.app = Accessing {0}
 # {0} is the name of the browser and {1} is the name of the devices OS that triggered the login
 idx.return.link.otponly.browser.on.os = {0} on {1}
-# {0} is the city and {1} is the state or country from where the login request originated
-idx.return.link.otponly.city.state.country = {0}, {1}
+# {0} is the city, {1} is the state, and {2} is the country from where the login request originated
+idx.return.link.otponly.city.state.country.all = {0}, {1}, {2}
+# This is used when 2 of {city, state, country} are populated: {0} is the city or state and {1} is the state or country from where the login request originated, respectively.
+idx.return.link.otponly.city.state.country.some = {0}, {1}
```

Comment:
Not an expert with denoting order, but we may want to further segment these into categories **if order changes** based on the input type.

@evelynteo-okta - Are you aware of any ordering changes between various languages for `City, State`, `State, Country`, or `City, Country`? I would assume the more focused area will always be the first positional argument, but am not 100% sure if this is correct.

Reply:
Do we even need i18n for country names.. I think we are passing whatever coming from server.. so server should have already localized it right ?

Reply:
@pradeepdewda-okta - We need to account for ordering of the geolocation, which needs to be handled by clients. [Not all languages](https://en.wikipedia.org/wiki/Chinese_punctuation) use commas for lists/separating values, so placing punctuation within our i18n resource files is required.

Reply:
@nikhilvenkatraman-okta - FYI ☝🏼

Reply:
@jmelberg-okta Great catch, looking forward to what the i18n team thinks

Reply:
@jmelberg-okta @nikhilvenkatraman-okta @justinabrokwah-okta @pradeepdewda-okta  `City, State, State, Country`, or `City, Country` --- **Yes, the ordering , groupings, styling (uppercase or lowercase) and punctuation of each piece of information, will change depending on the locale and country.** 

For example:
<img width="521" alt="Address-Format" src="https://user-images.githubusercontent.com/66045925/141003158-e8c0ceca-2388-4dd6-b699-b967b4ca7a43.png">

Postal codes, in countries that have them, are usually numeric, sometimes containing a space or a hyphen. Different countries use different terms for postal codes (e.g. Zone Improvement Plan, Postleitzahl) and different abbreviations for the terms (e.g. ZIP, PLZ, CEP, CAP, PIN).


Reference: 

International Component for Unicode (ICU) Address Format implementation at Salesforce: https://help.salesforce.com/s/articleView?id=sf.admin_supported_locales.htm&type=5

Reply:
Updated code to display location in these formats: `City, State, Country` or `City, Country`, preferring the former if the state is in the response